c++/src/ColumnReader.cc

```diff
@@ -828,7 +830,8 @@ namespace orc {
     std::vector<std::unique_ptr<ColumnReader>> children;
 
    public:
-    StructColumnReader(const Type& type, StripeStreams& stipe, bool useTightNumericVector = false);
+    StructColumnReader(const Type& type, StripeStreams& stipe, bool useTightNumericVector = false,
```

Comment:
I am thinking if we can use a new ColumnReaderOptions to include `useTightNumericVector` and `throwOnSchemaEvolutionOverflow` in case we will add more options. But that can be a separate patch since this is not public.